string cpu_model_name
float32[] cpu_utilizations
string[] cpu_frequencies
float32 cpu_temperature
float32 mem_usage_percentage